auto_nag/scripts/defectenhancementtask.py

```diff
@@ -78,20 +78,26 @@ def should_remove(bug):
 
     def get_bugs(self, date="today", bug_ids=[]):
         # Retrieve bugs to analyze.
-        bugs, probs = super().get_bugs(date=date, bug_ids=bug_ids)
+        bugs = super().get_bugs_from_backend(
+            "defectenhancementtask", date=date, bug_ids=bug_ids
+        )
         if len(bugs) == 0:
             return {}
 
         # Apply inverse transformation to get the type name from the encoded labels.
+        # TODO: Find a clean way to not need this
```

Comment:
Probably should move to the backend somehow

Reply:
We could add model specific data here: https://github.com/mozilla/bugbug/blob/master/http_service/models.py#L115

The component script also needs some extra data from the model: https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/component.py#L89

I think, as the data are not specific to a bug, to add a `get_extra_data` methods to both models in bugbug and the classification will call this method to get static extra_data to save in Redis and to returns to consumer, what do think @marco-c ?

Reply:
Returning this data from the backend sounds good to me.
My only doubt is whether to return it on every classification or with a specific API call.

Reply:
The extra HTTP endpoint is a good idea so we don't store a copy of the extra data per bug, will work on that

Reply:
For this specific case (defectenhancementtask), we can actually fix it by having an API call that returns the label names.
Or maybe we could even change the API to return something like {LABEL1: LABEL1_PROB, LABEL2: LABEL2_PROB, LABEL3: LABEL3_PROB} and so on instead of "prob", "index", "suggestion".

Reply:
Or: {"prob": {LABEL1: LABEL1_PROB, LABEL2: LABEL2_PROB, LABEL3: LABEL3_PROB}, "index": INDEX, "suggestion": SUGGESTION}

Reply:
I don't have a strong preference between all of these options

Reply:
The issue with the extra endpoint is that the web worker will need to start loading the models in memory, I will start with storing the extra data in Redis and we will monitor how much big the row will grow in Redis

Reply:
I'm adding the extra data in the saved row in Redis so it's the mininal changes and we can remove bugbug dependency in relman.

Reply:
Actually we cannot remove the bugbug dependency yet because the regressor scripts needs the model to classify commits and not bugs